tor/validation/formatting_validation.py

```diff
@@ -138,9 +145,14 @@ def check_for_heading_with_dashes(transcription: str) -> Optional[FormattingIssu
 
 def check_for_malformed_footer(transcription: str) -> Optional[FormattingIssue]:
     """Check if the transcription doesn't contain the correct footer."""
+    now = datetime.datetime.now()
+    pattern = FOOTER_PATTERN
+    if now.month == 4 and now.day in range(1, 5):  # some leeway from April 1st

```

Comment:
This should probably be a `datetime.now() in [timedelta range of day March 31st to April 2]`

Reply:
I like that a lot better too. Will adjust accordingly.